iOS Example Frame/iOS Example Frame/Storyboards/Base.lproj/Main.storyboard

```diff
@@ -397,21 +394,21 @@
                                                         <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                                         <color key="tintColor" white="0.0" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                                         <constraints>
-                                                            <constraint firstItem="m2u-mh-OW1" firstAttribute="leading" secondItem="HQk-AW-apP" secondAttribute="leading" id="LdG-Vw-NHL"/>
+                                                            <constraint firstItem="m2u-mh-OW1" firstAttribute="left" secondItem="HQk-AW-apP" secondAttribute="left" id="LdG-Vw-NHL"/>
```

Comment:
Do these changes break the RTL ? 

I see a lot of `leading and trailing` has been replaced by `left and right` in the nib. If this is intended, no problem, but if it wasn't we might need to make sure the RTL still works.

Reply:
Only Sample app main screen is supporting English LTR.